GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3015,46 +2978,7 @@ public void ChangeTerminalActiveFolder(string path)
                 return;
             }
```

Comment:
These lines have been moved to `ShellHelper`. Remove them, please.

Reply:
Remove the following lines, please. These checks are done by `_terminal.ChangeFolder`. No need to do them twice.
```
    if (_terminal?.RunningSession == null || string.IsNullOrWhiteSpace(path))
    {
        return;
    }
```

Reply:
Sorry, I must have missed these comments earlier. Fixed now.